### ISSUE TITLE
Not working

### ISSUE THREAD

**Issue by e36cuz** [discordtics/token-gen#8]
When i run it gives me this error

Traceback (most recent call last):
  File "C:\Users\XBOXB\OneDrive\Desktop\token-gen-main\token-gen-main\main.py", line 87, in <module>
    if choice == '1':
NameError: name 'choice' is not defined

**Comment by Chris1238** [comment]
It's a virus

**Comment by Daamss** [comment]
yeah this is a virus that steals all your info lol. Do not download
